app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+
+  scope :recent_three, -> { :posts.limit(3).order(created_at: :desc) }
```

Comment:
- [ ] Please implement the previous reviewer's suggestion regarding creating a new method that returns the 3 most recent posts like this: 
``` ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```